parasl/parasl.txt

```diff
@@ -82,7 +82,7 @@ s2.x = repeat(0, 5);
 
 Function entity is an abstraction mechanism. It is a callable.
 
-f0 = { v3 = input(0) + 1; v3 * v3; };
+f0 : () = { v3 = input(0) + 1; v3 * v3; };
```

Comment:
I think we can make rule like: block outside of if/while/layer context is always callable. Then example is correct:

f0 = { v3 = input(0) + 1; v3 * v3; };
v0 = { v3 = input(0) + 1; v3 * v3; }(1);
v1 = f0(1);

etc...

Reply:
`f0 = { v3 = input(0) + 1; v3 * v3; };`
And what about inputs? Here we create function w/ 1 argument (`input(0)`), but we do not forward any arguments in `(...)`. I think it is strange.

Reply:
Also, we do not have any declarations/other_code outside layer0. It contradicts the rule given. As result, we can not write `f = {...};` at all. 
On the other side, excluding using of `{...}` inside layer exclude lambda syntax.

Reply:
> Also, we do not have any declarations/other_code outside layer0. It contradicts the rule given. As result, we can not write `f = {...};` at all.

I don't quite get what is wrong with declarations/syntax inside layer-0 callable block?